extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -453,16 +525,26 @@ class CornerstoneViewportService implements IViewportService {
       !hangingProtocolService.customImageLoadPerformed
     ) {
       // delegate the volume loading to the hanging protocol service if it has a custom image load strategy
-      return hangingProtocolService.runImageLoadStrategy({
-        viewportId: viewport.id,
-        volumeInputArray,
-      });
+      if (
+        hangingProtocolService.runImageLoadStrategy({
+          viewportId: viewport.id,
+          volumeInputArray,
+        })
```

Comment:
`runImageLoadStrategy` returns `void`, what is the boolean here to check? Discard this comment if you have changed `runImageLoadStrategy` (I haven't read that part yet)